### PR TITLE
feat: [PROD-12490] add DatasetMetadata component

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: src/views/DatasetManager/components/DatasetMetadata/DatasetMetadata.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        <Grid item>
+          <TagsEditor
+            id="dataset-tags"
+            labels={tagsEditorLabels}
+            values={dataset?.tags}
+            readOnly={false}
+            onChange={(newTags) => {
+              // TODO: add saga & call back-end to update dataset tags
+              // console.log('New tags:');
+              // console.log(newTags);
```
Comment (esasova) [inline src/views/DatasetManager/components/DatasetMetadata/DatasetMetadata.js:93]: logs can be deleted (lines 102-103 as well) :slightly_smiling_face:

File: src/views/DatasetManager/components/DatasetMetadata/DatasetMetadata.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Card, Grid, IconButton, Paper } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import FileCopyOutlinedIcon from '@mui/icons-material/FileCopyOutlined';
+import { TagsEditor } from '@cosmotech/ui';
+import { useDatasetMetadata } from './DatasetMetadataHook';
+import { DescriptionEditor, MetadataItem } from './components';
+import { ApiUtils } from '../../../../utils/ApiUtils.js';
```
Comment (esasova) [inline src/views/DatasetManager/components/DatasetMetadata/DatasetMetadata.js:12]: we can import it directly from _/utils_ :slightly_smiling_face:

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: src/views/DatasetManager/components/DatasetMetadata/DatasetMetadata.js (hunk 2 above)
Comment (csm-thu) [reply src/views/DatasetManager/components/DatasetMetadata/DatasetMetadata.js:12]: Nice catch, thanks :smile:

**Pull request merged**